app/src/lib/api.ts

```diff
@@ -1233,6 +1233,45 @@ export class API {
     return false
   }
 
+  /**
+   * Re-run all of the failed jobs and their dependent jobs in a workflow run
+   * using the id of the workflow run.
+   */
+  public async rerunFailedJobs(
+    owner: string,
+    name: string,
+    workflowRunId: number
+  ): Promise<boolean> {
+    const path = `/repos/${owner}/${name}/actions/runs/${workflowRunId}/rerun-failed-jobs`
+    const response = await this.request('POST', path)
+    try {
+      return response.ok
+    } catch (err) {
+      log.debug(
+        `Failed to rerun failed workflow jobs for (${owner}/${name}): ${workflowRunId}`
+      )
+    }
```

Comment:
Response.ok is just a plain old property, accessing it can never throw, I'm guessing you meant to guard again `this.request` throwing?

```suggestion
return this.request('POST', path)
      .then(x => x.ok)
      .catch(err => {
        log.debug(
          `Failed to rerun failed workflow jobs for (${owner}/${name}): ${workflowRunId}`,
          err
        )
        return false
      })
```